modules/web-console-cso.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * web_console/scan-from-web-console.adoc
+
+[id="web-console-cso_{context}"]
+= Run Container Security Operator in {product-title}
+
+To start the Container Security Operator, go to the {product-title}
+web console, then select and install that Operator from the Operator Hub,
+as described here.
+
+.Prerequisites
+
+* Be running {product-title} 4.2 or later
+* Have administrator privileges to the {product-title} cluster
+* Be running containers that come from a Red Hat Quay or Quay.io registry
+
+.Procedure
+
+To start using the CSO in OpenShift, do the following:
+
+. Go to Operators -> OperatorHub (select Security) to see the available `Container Security` Operator.
+
+. Select the `Container Security` Operator, then select `Install`
+to go to the Create Operator Subscription page.
+
+. Check the settings (all namespaces and automatic approval strategy, by default), and select
+`Subscribe`. The `Container Security` Operator appears after a few moments on the `Installed Operators` screen.
+
+. Optionally, you can add custom certificates to the CSO. In this example, create a certificate
+named quay.crt in the current directory. Then run the following command to add the cert to the CSO (restart
+the Operator pod for the new certs to take effect):
++
+```
+$ oc create secret generic container-security-operator-extra-certs --from-file=quay.crt -n openshift-operators
+```
+
+. Open the OpenShift Dashboard (Home -> Overview). A link to
+_Quay Image Security_ appears under the status section, with a listing of the number
+of vulnerabilities found so far. Select the link to see a _Quay Image Security breakdown_, as shown in the following figure:
++
+image:image_security.png[Access image scanning data from {product-title} dashboard]
+
+. You can do one of two things at this point to follow up on any detected vulnerabilities:
++
+*  Select the link to the vulnerability. You are taken to the container
+registry, Red Hat Quay, Quay.io, or other registry where the container came
```

Comment:
Suggest removing ", Red Hat Quay, Quay.io, or other registry". Later in the paragraph you give Quay.io as an example

Reply:
Done.